cortex/secondary/call_duration.py

```diff
@@ -2,14 +2,16 @@
 import numpy as np
 
 from ..feature_types import secondary_feature, log
-from ..raw.calls import calls
+from ..raw.telephony import telephony
 
 MS_IN_A_DAY = 86400000
+
+
 @secondary_feature(
     name='cortex.feature.call_duration',
-    dependencies=[calls]
+    dependencies=[telephony]
 )
-def call_duration(incoming=True, **kwargs):
+def call_duration(incoming, **kwargs):
     """The time (in ms) spent talking on the phone.
 
     Args:
```

Comment:
Make sure to edit the docstring to reflect the updated function of new parameters